content/management/configuration-api/v3.3/index.mdx

```diff
@@ -3678,6 +3678,136 @@ curl -X POST \
+
+#### Request
+
+| Parameter         | Required | Data type | Notes                                                |
+| ----------------- | -------- | --------- | ---------------------------------------------------- |
+| `owner_client_id` | No       | `string`  | Required when authorizing via PAT; ignored otherwise |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.3/configuration/action/enable_webhooks \
+  -H 'Content-Type: application/json' \
+  -d '{}'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Disable Webhooks
+
+Disables webhooks assigned to **Client Id** from authorization token.
+
+#### Specifics
+
+|                 |                                                                          |
+| --------------- | ------------------------------------------------------------------------ |
+| Method URL      | `https://api.livechatinc.com/v3.3/configuration/action/disable_webhooks` |
+| Required scopes | `webhooks.state:rw`                                                      |
+
+#### Request
+
+| Parameter         | Required | Data type | Notes                                                |
+| ----------------- | -------- | --------- | ---------------------------------------------------- |
+| `owner_client_id` | No       | `string`  | Required when authorizing via PAT; ignored otherwise |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.3/configuration/action/disable_webhooks \
+  -H 'Content-Type: application/json' \
+  -d '{}'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Get Webhooks State
+
+Gets state of webhooks assigned to **Client Id** from authorization token.
+
+#### Specifics
+
+|                 |                                                                            |
+| --------------- | -------------------------------------------------------------------------- |
+| Method URL      | `https://api.livechatinc.com/v3.3/configuration/action/get_webhooks_state` |
+| Required scopes | `webhooks.state:ro`                                                        |
+
+#### Request
+
+| Parameter         | Required | Data type | Notes                                                |
+| ----------------- | -------- | --------- | ---------------------------------------------------- |
+| `owner_client_id` | No       | `string`  | Required when authorizing via PAT; ignored otherwise |
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.3/configuration/action/get_webhooks_state \
+  -H 'Content-Type: application/json' \
+  -d '{}'
+```
+
+</CodeSample>
+
+<CodeResponse type="configuration" version="v3.3" json="getWebhooksState"/>
```

Comment:
It doesn't get rendered.
![image](https://user-images.githubusercontent.com/42515529/103534774-d376cc00-4e8f-11eb-9f6e-ea9910aca571.png)

You need to export getWebhooksState in `payloads/management/v3.3/configuration-api/index.js`